### ISSUE TITLE
confusing "user dialog" item

### ISSUE THREAD

**Issue by adriatic** [aurelia/cli#188]
The command `au new` results with the first "dialog" question:
```
au new
                      _ _          ____ _     ___
  __ _ _   _ _ __ ___| (_) __ _   / ___| |   |_ _|
 / _` | | | | '__/ _ \ | |/ _` | | |   | |    | |
| (_| | |_| | | |  __/ | | (_| | | |___| |___ | |
 \__,_|\__,_|_|  \___|_|_|\__,_|  \____|_____|___|


What would you like to name your new project?
```

Since the most likely answer to this question is `yes` - you actually get your project named `yes` :-).

Most of code generators form this question like this 

```
Enter the name of your new project [new-project] > 
```

where the string in the angular brackets represents the default value.

**Comment by jsobell** [comment]
Well, it does say *what* would you like to name it :)

**Comment by EisenbergEffect** [comment]
We could rephrase it to say "Please enter the name of your new project." That's pretty easy to do I suppose.

**Comment by adriatic** [comment]
@jsobell - you are quite right, I completely missed the first word of that question - 'What'.

Despite that I believe that Rob's rephrasing is better English.

**Comment by martindavid** [comment]
I agree with @adriatic at least we can give user default name for the project. A lot of generator do that.

**Comment by MCorzo** [comment]
i think would be great if the cli show a proper message when "au run" is executed in a folder without a aurelia project inside, now only return the message "Invalid Command: run".
